shell/assets/styles/themes/_dark.scss

```diff
@@ -29,19 +29,84 @@
   $disabled: $light;
 
   //Contrast colors
-  $contrasted-dark: $lightest !default;
+  $contrasted-dark: $grey-0 !default;
   $contrasted-light: $darkest !default;
 
   $drag-over: #DCDEE7;
+  $body-bg: #2F2F42;
+
+  /*********************/
+  
+  --body__bg: #{$body-bg};
```

Comment:
We had an out of band discussion on the introduction of new css variables like `--body__bg`. Widespread usage of variables like this could cause backwards compatibility issues with extensions that are built with these new variables in place, as they will not exist in previous versions of Rancher Dashboard.